### ISSUE TITLE
Visualize test results ?

### ISSUE THREAD

**Issue by moonsh** [YanFangCS/CyCTR-Pytorch#1]
Hi ! thank you for sharing your work.

Is there an easy way to visualize results with the pre-trained model ?

**Comment by GengDavid** [comment]
Hi @moonsh, the results can be seen by saving or showing the [output](https://github.com/YanFangCS/CyCTR-Pytorch/blob/218c14967d1c1fe0060b4463550aaf2a99e017dc/test.py#L208) in the test script. Note that it may contain paddings on the border. You could remove them according to the label.

**Comment by moonsh** [comment]
I see. Thank you ! @GengDavid
